lib/index.js

```diff
@@ -305,20 +305,20 @@ Queue.prototype._reclaim = function(id) {
     queue: other.get(this.keys.QUEUE) || []
   };
 
-  // add their queue to ours, resetting run-time to immediate and attempt# to 0
+  // add their queue to ours, resetting run-time to immediate and copying the attempt#
   each(function(el) {
     our.queue.push({
       item: el.item,
-      attemptNumber: 0,
+      attemptNumber: el.attemptNumber,
       time: self._schedule.now()
     });
   }, their.queue);
 
-  // if the queue is abandoned, all the in-progress are failed. retry them immediately and reset the attempt#
+  // if the queue is abandoned, all the in-progress are failed. retry them immediately and increment the attempt#
   each(function(el) {
     our.queue.push({
       item: el.item,
-      attemptNumber: 0,
+      attemptNumber: el.attemptNumber + 1,
```

Comment:
why increment? rather than keep it the same?

Reply:
I did this because of this statement :
> if the queue is abandoned, all the in-progress are failed

Given that they're failed I thought their attempt number should be incremented. Not really sure if that make sense, let me know your thoughts

Reply:
SGTM!